config/components/rbac/role.yaml

```diff
@@ -347,3 +347,21 @@ rules:
   - get
   - patch
   - update
+- apiGroups: [ "apiextensions.k8s.io" ]
+  resources: [ "customresourcedefinitions" ]
+  verbs: [ "get", "list", "watch", "create", "update", "delete" ]
```

Comment:
Why do we need this?

Reply:
I think the "read" rbac might be needed by Kueue to be able to watch the CRD definitions. However, almost certainly we don't need the "write" rbac - create update and delete.

Also, ideally, these should be genereted based on kube-builder marker comments, rather than specified manually, [example](https://github.com/kubernetes-sigs/kueue/blob/e9b35497ccf5b0534cce64a2a5f71c81b0926d6d/pkg/controller/jobs/job/job_controller.go#L72-L81)